.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.activeBackground": "#e7f3fb",
+        "activityBar.background": "#e7f3fb",
+        "activityBar.foreground": "#15202b",
+        "activityBar.inactiveForeground": "#15202b99",
+        "activityBarBadge.background": "#e876bb",
+        "activityBarBadge.foreground": "#15202b",
+        "commandCenter.border": "#15202b99",
+        "sash.hoverBorder": "#e7f3fb",
+        "statusBar.background": "#bbdcf4",
+        "statusBar.foreground": "#15202b",
+        "statusBarItem.hoverBackground": "#8fc5ed",
+        "statusBarItem.remoteBackground": "#bbdcf4",
+        "statusBarItem.remoteForeground": "#15202b",
+        "titleBar.activeBackground": "#bbdcf4",
+        "titleBar.activeForeground": "#15202b",
+        "titleBar.inactiveBackground": "#bbdcf499",
+        "titleBar.inactiveForeground": "#15202b99"
+    },
```

Comment:
Do I have to use your positron colours too? 😆

Reply:
They are lovely, no?

Reply:
I've deleted for now since it really doesn't work with my dark theme 😞 But we can figure something out if you really want this.